core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -3884,7 +3884,7 @@ public void _testWinPartClause() {
 
     winSql(
         "select cume_dist() over w , ^rank()^\n"
-        + "from emp \n"
+        + "from emp group by deptno\n"
```

Comment:
There shouldn't be a need to add group-by here since group-by is only needed when you have nested aggregate.  This points to some other potential issue.